integration_test/tests/src/tests/HubManager.test.ts

```diff
@@ -5,6 +5,9 @@ import { HubManagerPage, DFSPRow } from '../page-objects/pages/HubManager';
 import chance from 'chance';
 import { Selector } from 'testcafe';
 
+// NOTE: There are added `t.wait` functions on `t.clicks` to give the UI time to
+//       update. This is to account for observed flakiness with test-cafe
+//       tests.
```

Comment:
It would be useful to elaborate slightly on flakiness, if possible. Something like "it was observed that tcafe clicked the wrong element" or "tcafe seemed to click before the page loaded" or "tcafe seemed to fail the test too quickly when it didn't detect the element". That said, I know it's sometimes very difficult to elaborate on flakiness, because if you knew what the problem was, you might stand a chance of fixing it.

Reply:
done.